src/opera/api/service/sqldb_service.py

```diff
@@ -63,7 +63,7 @@ def get_deployment_log(self, blueprint_token: str = None, session_token: str = N
         """
         pass
 
-    def save_git_transaction_data(self, blueprint_token: uuid, version_tag: str, revision_msg: str, job: str,
+    def save_git_transaction_data(self, blueprint_token: uuid, project_domain: str, version_tag: str, revision_msg: str, job: str,
```

Comment:
What is the purpose of project_domain arg? Also, it does not appear in any of Overriding Methods

Reply:
No need. At first I was thinking adding domain to git transaction, but realized it is a bad Idea. Was pretty sure I reversed everything.